src/common/type-checks.ts

```diff
@@ -20,6 +20,10 @@ export const isBool = (val: unknown): val is boolean => {
   return typeof val === "boolean"
 }
 
+export function isCryptoKey(val: unknown): val is CryptoKey {
+  return hasProp(val, "algorithm") && hasProp(val, "extractable") && hasProp(val, "type")
```

Comment:
Should this just be
```suggestion
  return val instanceof CryptoKey
```
?

Reply:
Probably yeah 😄 Don't remember why I did it this way.

Reply:
Undoing this change, for some reason the tests freeze with this 🤨 
https://github.com/fission-codes/webnative/actions/runs/3541918255/jobs/5946780663